### ISSUE TITLE
Linux Compatibility

### ISSUE THREAD

**Issue by cactushydrocodone** [diantahoc/chan-archiver#9]
Hey,

Is it possible to add Linux compatibility on this one?
I can compile it on linux, but the Download function wont work...

Is there a way to fix this?

Greetings

**Comment by diantahoc** [comment]
Do you mean download as zip function?

**Comment by cactushydrocodone** [comment]
Nope, i mean the whole app doesnt work well on linux, i can compile it and start it, but the download function of the APP doesnt work.

**Comment by diantahoc** [comment]
I'll give it a look.

**Comment by cactushydrocodone** [comment]
Update on this?

**Comment by diantahoc** [comment]
Not yet, I have been busy this while. 
Don't loose hope.

**Comment by cactushydrocodone** [comment]
Can i loose hope now? :p

**Comment by diantahoc** [comment]
No, stay alive.

**Comment by cactushydrocodone** [comment]
can i help you something with this?

**Comment by cactushydrocodone** [comment]
Your project works out of the box on the newest mono runtime.
**to run it, you have to compile it on a windows machine with for example visual studio, then copy the executable and ALL files in the debug folder (the libraries to!) into a folder, zip it, and drop it on your linux machine**
**you have to disable HTTPS to make it work, somehow with https enabled, the console gets spammed with failed requests and it crashes the application**
You have to install the following Packages (on ubuntu) to make it fully work:

- ca-certificates-mono (THIS IS IMPORTANT!!)
- mono-complete
- referenceassemblies-pcl

**Comment by diantahoc** [comment]
Hello

First I apologize for this huge delay, it was a busy year and sometimes honestly I forgot about this. Today I finally got the chance to investigate the issue, so I compiled the project on openSUSE using Mono, but the download button worked. I jumped to this thread to ask you more details and I saw your comment.

Thank you for your patience first, and your persistence to investigate the problem. 

On the other hand, I will see if I can make the HTTPS problem easier. I will also include the missing libraries in git so that you wont need to compile it first on Windows, and perhaps update the readme.

Happy archiving.
